### PR TITLE
READY : Change test routine `startNjsWithReadyDelayStructural*`

### PR DESCRIPTION
**Pull request by JackieJoo**
1. Changed & extended test routine `startNjsWithReadyDelayStructural`.
2. Changed & extended test routine `startNjsWithReadyDelayStructuralMultiple`.

```
function startNjsWithReadyDelayStructural( test )
/* qqq for Yevhen : add varying `sync` and `deasync`
   do not combine `detaching` with `dry`
   do not combine `detaching` with `sync` and `deasync`
   */
 /* qqq for Yevhen : add varying `sync` and `deasync` and `dry` for test routine startNjsWithReadyDelayStructuralMultiple. */
```

### CODE REVIEW

File: proto/wtools/abase/l4_process.test/Execution.test.s [hunk 1]
```diff
@@ -16625,8 +16938,141 @@ function startNjsWithReadyDelayStructuralMultiple( test )
...
+  //   test.identical( options.conTerminate.exportString(), 'Consequence:: 0 / 0' );
+  //   test.identical( options.conDisconnect, null );
+  //   test.identical( options.conStart.exportString(), 'Consequence:: 0 / 0' );
+
+  //   return returned;
+  // })
+
 }
 
+startNjsWithReadyDelayStructuralMultiple.timeOut = 120000;
```
Comment (Wandalen) [inline proto/wtools/abase/l4_process.test/Execution.test.s:17075]: - use 3e5 format.
- make sure time is enough fpr the slowest machine

Comment (JackieJoo) [reply proto/wtools/abase/l4_process.test/Execution.test.s]: Updated timeout to be `18e4`. For 3 runs, no timeout needed, so 180000 must be enough for 9 runs.

File: proto/wtools/abase/l4_process.test/Execution.test.s [hunk 2]
```diff
@@ -17080,8 +17393,141 @@ function startNjsWithReadyDelayStructuralMultiple( test )
...
+  //   test.is( options.streamOut !== options.streamErr );
+  //   test.identical( options.ready.exportString(), 'Consequence:: 0 / 3' );
+  //   test.identical( options.conTerminate.exportString(), 'Consequence:: 0 / 0' );
+  //   test.identical( options.conDisconnect, null );
+  //   test.identical( options.conStart.exportString(), 'Consequence:: 0 / 0' );
+
+  //   return returned;
+  // })
+
 }
```
Comment (Wandalen) [inline proto/wtools/abase/l4_process.test/Execution.test.s:17529]: rapidity?

Comment (JackieJoo) [reply proto/wtools/abase/l4_process.test/Execution.test.s:17531]: Added.

**Pull request merged**